src/services/crypto-compare.js

```diff
@@ -17,14 +19,16 @@ const requestFactory = {
 }
 
 const requestPool = genericPool.createPool(requestFactory, {
-  max: MAX_REQUEST_PER_SECOND
+  max: MAX_RESOURCES
 })
 
 class CryptoCompareService {
   async get (url, options) {
     const client = await requestPool.acquire()
     const response = await client.get(url, options)
 
+    await sleep(100)
```

Comment:
Could you add a comment describing what this sleep is for?

Reply:
I believe you are on an outdated commit :susp: I tried to use that to get around the api limit but it has since been removed.

Reply:
yeah I was going commit by commit to only see the new changes 😂

Reply:
that will work from https://github.com/ArkEcosystem/explorer/pull/708/commits/53a988ec12f918a0db8d90c92923f04f872dcb5e onward 😂